src/main/java/Duke.java

```diff
@@ -1,10 +1,255 @@
+        System.out.println(HORIZONTAL_LINE);
+        System.exit(0);
+    }
+
+    public static void printList(Task[] listOfInputs, int inputCounter) {
+        for (int i = 0; i < inputCounter; i++) {
+            System.out.println(i + 1 + "." + listOfInputs[i].toString());
+        }
+    }
+
+    // Returns a string containing taskName
+    public static String todoHandler(String userInput) {
+
+        // Format of userInput: <command> <task_name>
+        int numberOfWords = userInput.split(" ").length;
+        String[] userInputArray = new String[numberOfWords - 1];
+        String taskName = "";
+
+        userInputArray = userInput.split(" ");
+
+        // Remove to_do command from the finalString
+        for (int i = 1; i < numberOfWords; i++) {
+            taskName += userInputArray[i] + " ";
+        }
+
+        // Remove trailing space at the end of taskName
+        taskName = taskName.substring(0, taskName.length() - 1);
+
+        return taskName;
+
+    }
+
+    // Returns an array containing [taskName, deadline]
+    public static String[] deadlineHandler(String userInput) {
+
+        // Format of userInput: <command> <taskName> /by <deadline>
+        int numberOfWords = userInput.split(" ").length;
+        int endOfTaskNameIndex = 2;
+        String taskName = "";
+        String deadline = "";
+        String[] userInputArray = new String[numberOfWords];
+        String[] cleanedUserInputArray = new String[numberOfWords - 1];
+        String[] finalArray = new String[2];
+        boolean isByPassed = false;
+
+        userInputArray = userInput.split(" ");
+
+        // Remove "/by" and put each word into cleanedUserInputArray
+        for (int i = 0; i < numberOfWords; i++) {
+            if (userInputArray[i].equals("/by")) {
+                endOfTaskNameIndex = i;
+                isByPassed = true;
+                continue;
+            }
+            if (!isByPassed) {
+                cleanedUserInputArray[i] = userInputArray[i];
+            } else {
+                cleanedUserInputArray[i - 1] = userInputArray[i];
+            }
+        }
+
+
+        // Collate words for the taskName
+        for (int i = 1; i < endOfTaskNameIndex; i++) {
+            taskName += cleanedUserInputArray[i] + " ";
+        }
+
+        // Remove trailing space at the end of taskName
+        taskName = taskName.substring(0, taskName.length() - 1);
+        finalArray[0] = taskName;
+
+        // Collate words for the deadline
+        for (int i = endOfTaskNameIndex + 1; i < numberOfWords; i++) {
+            deadline += userInputArray[i] + " ";
+        }
+
+        // Remove trailing space at the end of deadline
+        deadline = deadline.substring(0, deadline.length() - 1);
+        finalArray[1] = deadline;
+
+        return finalArray;
+    }
+
+    // Returns an array containing [taskName, from, to]
+    public static String[] eventHandler(String userInput) {
```

Comment:
eventHandler Method is also pretty long as it exceeds 30 lines of code. Shortening it and reusing code from before might be better